unity-package-with-correct-tests/com.fake.notarealpackage/Tests/Runtime/fake.notarealpackage.RuntimeTests.asmdef

```diff
@@ -0,0 +1,11 @@
+{
+    "name": "fake.notarealpackage.RuntimeTests",

```

Comment:
Prefer `example.com` or `com.example` instead of using actual website names.

That domain is meant for such uses specifically.

Reply:
Good call! I've addressed this in my most recent commit - the package is now called `com.example.testpackage`.

It has ~40 changes, but that's mainly just because I renamed the folder for the package to the new package name. The key changes are:

- Renamed the package folder itself
- Updated the package.json with the new package name & updating the info in it to better match the new name
- Updated all the assembly definitions to match the new name
- Updated the `main.yml` tests to point at the new package name

If that was the only note, then I have nothing else to add 👍

Reply:
Cool. Looks good to me. 

Probably good if some of @davidmfinol, @AndrewKahr, @frostebite, @GabLeRoux, @lazerwalker could have a look. And then hopefully we can release it soon :)

Reply:
What a journey! I just caught up with the history on this one. I don't see any major issues with the limitations, `jq` is a pretty common requirement, using only Linux runners for the time-being seems fine for a initial release of a new feature.

Seems like there have already been several users who have used the fork successfully with all issues reported already having been addressed.

Great work!